org.adempiere.ui.zk/WEB-INF/src/org/adempiere/webui/apps/ProcessParameterPanel.java

```diff
@@ -951,6 +954,8 @@ private void dynamicDisplay() {
 				}
 			} else if (editor.isVisible()) {
 				editor.setVisible(false);
+				m_Divs.get(i).getParent().setVisible(false);
```

Comment:
While this looks fine, it is a rather strange way of doing thing here. I means if you want to make Row (i.e Div.getParent()) visible/invisible, why don't you just keep a list of Row instead of Div ?

Reply:
True, first I couldn't make it work with Row, but I've just realised I was making the List of Row incorrectly. I see, what I need is already there..